pkg/sdk/testint/tags_integration_test.go

```diff
@@ -16,9 +16,7 @@ func TestInt_Tags(t *testing.T) {
 	client := testClient(t)
 	ctx := context.Background()
 
-	databaseTest, databaseCleanup := testClientHelper().Database.CreateDatabase(t)
-	t.Cleanup(databaseCleanup)
-	schemaTest, schemaCleanup := testClientHelper().Schema.CreateSchemaInDatabase(t, databaseTest.ID())
+	schema, schemaCleanup := testClientHelper().Schema.CreateSchema(t)
```

Comment:
I think tags could also be tested on the predefined schema, but I guess that can be adjusted later on during the refactor for tags.

Reply:
Yep, it's not the scope for this change (I didn't want to think about it for every resource's test in this change)